src/components/AboutUs/AboutUs.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
```

Comment:
AboutUs is a page. So, I propose to create a folder called `pages` and move this component in that folder.